hub/constraints/tx_skip/constraints.lisp

```diff
@@ -11,16 +11,16 @@
      (- HUB_STAMP (prev HUB_STAMP))))
 
 (defconst 
-  tx-skip---sender-account---row-offset       0
-  tx-skip---recipient-account---row-offset    1
-  tx-skip---coinbase-account---row-offset     2
-  tx-skip---transaction-row---row-offset      3)
+  tx-skip---row-offset---sender-account       0
+  tx-skip---row-offset---recipient-account    1
+  tx-skip---row-offset---coinbase-account     2
+  tx-skip---row-offset---transaction-row      3)
```

Comment:
Ignore this @letypequividelespoubelles : it's a more coherent renaming of row offset constants.